Topic1/src/com/topic1/MostExpensive.java

```diff
@@ -0,0 +1,31 @@
+package com.topic1;
+
+import java.util.HashSet;
+import java.util.Iterator;
+
+public class MostExpensive implements FindItem {
+
+	HashSet<Item> items;
```

Comment:
[Medium] Please remember to set the Access modifier. Also, you don't need to store this information. You could send it as a parameter to your *find* method